src/site/content/en/fast/identify-slow-third-party-javascript/index.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: post
+title: Identify slow third-party JavaScript
+subhead: Supercharge your performance detective skills with Lighthouse and Chrome DevTools.
+authors:
+  - mihajlija
+date: 2019-07-16
+# Add an updated date to your post if you edit in the future.
+# updated: 2019-06-27
+hero: hero.jpg
+alt: Inspecting a laptop with a magnifying glass.
+# You can adjust the position of your hero image with this property.
+# Values: top | bottom | center (default)
+# hero_position: bottom
+description: |
+  Learn how to use Lighthouse and Chrome DevTools to identify slow third-party resources.
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - fast
+---
+
+<style>
```

Comment:
Since this icon is used as an instruction would you mind adding it to the Instruction widget, similar to the preview step here: https://github.com/GoogleChrome/web.dev/blob/fac8e165ecdb0b8e2c92fd18e52e37bc5ee9da7e/src/site/_includes/components/Instruction.js#L76

Also suggest using an svg image so it looks nice and sharp on retina displays.

Reply:
Is adding this to the instruction widget a to-do for me? (I still have that Instruction updates PR open...)

Reply:
Oh I was suggesting Milica do it since she needs to track down the svg icon and style it.

Reply:
@mfriesenhahn Have you grabbed this icon from DevTools docs? I think it's just a screenshot from DevTools, hence the resolution.  https://developers.google.com/web/tools/chrome-devtools/evaluate-performance/imgs/reload-page.png 

If we want the authentic DevTools reload icon, someone has to get it from the DevTools team. Alternatively, this is the closest free SVG I could find online, but the licence kinda requires we add attribution: https://thenounproject.com/search/?q=reload&i=1909551

Reply:
There's a material design refresh icon svg we could use:
https://material.io/tools/icons/?icon=refresh&style=baseline

Inspecting the devtools itself it looks like they use a .png and have a 2x version. I'm trying to track down where I can download it.

Reply:
I grabbed that PNG from a /web doc about DevTools, yes. I assumed that was canonical, but if there's an SVG to be had, great!

Reply:
SVG has been located! https://github.com/ChromeDevTools/devtools-frontend/blob/master/front_end/Images/src/largeIcons.svg